tests/Configurator/CopyDirectoryFromPackageConfiguratorTest.php

```diff
@@ -115,17 +114,18 @@ private function cleanUpTargetFiles()
     }
 
     /**
-     * Courtesy of http://php.net/manual/en/function.rmdir.php#98622
+     * Courtesy of http://php.net/manual/en/function.rmdir.php#98622.
      */
-    private function rrmdir($dir) {
+    private function rrmdir($dir)
+    {
         if (is_dir($dir)) {
             $objects = scandir($dir);
             foreach ($objects as $object) {
-                if ($object != "." && $object != "..") {
-                    if (filetype($dir."/".$object) == "dir") {
-                        $this->rrmdir($dir."/".$object);
+                if ('.' != $object && '..' != $object) {
+                    if ('dir' == filetype($dir.'/'.$object)) {
```

Comment:
Isn't it possible to use `===` in these statements instead?

Reply:
Sure, but out of scope of this PR. This PR is only about applying changes from running `php-cs-fixer`.